src/main/java/com/google/codeu/data/User.java

```diff
@@ -0,0 +1,20 @@
+package com.google.codeu.data;
+
+public class User {
+
+    private String email;
+    private String aboutMe;
+
+    public User(String email, String aboutMe) {
+        this.email = email;
```

Comment:
It might make sense to check to see if email is a null value (and throw an exception if it is.)

You could be even more strict, and require that it at least looks like a real email address, but I'll leave that up to you.